src/backend/context.rs

```diff
@@ -700,6 +733,19 @@ impl Context {
 			Event::WindowEvent(WindowEvent::Resized(event)) => {
 				let _ = self.resize_window(event.window_id, event.size);
 			},
+			Event::WindowEvent(WindowEvent::MouseWheel(event)) => {
+				let delta = match event.delta {
+					winit::event::MouseScrollDelta::LineDelta(_, y) => y,
+					_ => 0.0
+				};
+				let _ = self.zoom_window(event.window_id, delta * 0.01);
```

Comment:
This now zooms around the origin. It would be very nice too zoom around the cursor position instead. That isn't really possible with the current `zoom_window`/`pan_window` API though, so I think we need something more flexible theret.

Reply:
I was under the impression that zoom occurs around the origin but seems like not. I will change it.

Reply:
It does, but the origin is probably the top left corner, depending on when you apply the zoom. In any case, zooming around the cursor is much nicer.